### ISSUE TITLE
Can't play Internet Videos (eg Youtube) videos in Unicord

### ISSUE THREAD

**Issue by Delstrom2** [UnicordDev/Unicord#83]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When A Youtube or other internet video url has been pasted in a channel and previews in the feed, when you click the play icon it loads and plays the video in the app

### Expected Behavior

When I click play, the video preview turns grey like as if it is loading, but never loads

### Steps To Reproduce

1: Open Unicord 
2: Paste Youtube URL into a channel and wait for it to embed
3: Click Play

### Screenshots

<img width="456" alt="image" src="https://user-images.githubusercontent.com/78463422/147604616-010ea2b1-d54a-42d0-a953-cf9a055c89e0.png">

### Version

1.7.6

### Windows Version

Windows 11 (22000)

### Platform?

Desktop

**Comment by Kotori7** [comment]
I can't reproduce this; it works for me with both kinds of YouTube URL formats
![image](https://user-images.githubusercontent.com/26189638/147851511-7216b8c2-a0f5-4af2-8b4e-85d0b3b4ba3f.png)

**Comment by Delstrom2** [comment]
Is there a log file of some sort I could try to send you? For some extra context, it's running on an arm windows device, but I don't believe any x86 emulation would be the problem if it isn't compiled for arm.

**Comment by Delstrom2** [comment]
I apologize that I didn't give you the context of what kind of machine I was on earlier. I extracted the arm, x86 and x64 appx binaries from the release 1.7.6 appx file with 7zip and tested each of them. Neither the arm nor the x86 versions of unicord were able to play the youtube videos on my machine, but for some reason the x64 version didn't have any issues with it. I modified the title to be more representative of what I've found.
